axelrod/tournament.py

```diff
@@ -27,6 +27,8 @@ def __init__(self, players, tournament_type=round_robin, name='axelrod',
             an axelrod Match object
         name : string
             A name for the tournament
+        clone_opponents: boolean
```

Comment:
Can we give more context here, I'm actually not sure I understand what is going on so for a beginner using the library I think this would certainly be confusing.